ooniprobe/AppDelegate.mm

```diff
@@ -25,7 +25,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     application.statusBarStyle = UIStatusBarStyleLightContent;
     
-    [self registerNotifications];
+    //TODO removed notification popup on start. TO decide where to put it
+    //[self registerNotifications];
```

Comment:
Why?

Reply:
Because will be prompt after the first test ran in the new app

Reply:
Wouldn't this mean that the first test's results will be collected without informed consent? Wouldn't this expose users to risks like running a test without knowing its implications? I am against doing that, TBH.

Reply:
It's Notification popup, not informed consent

Reply:
Yeah, sorry, also @hellais pointed this out on Slack. I really misunderstood the context!